src/main/java/sirius/biz/web/BizController.java

```diff
@@ -256,11 +257,19 @@ protected void load(WebContext ctx, BaseEntity<?> entity, List<Mapping> properti
             Property property = entity.getDescriptor().getProperty(columnProperty);
             String propertyName = property.getName();
 
+            // If the parameter is not present in the request we just skip it to prevent resetting the field to null
+            final Value parameterValue = ctx.get(propertyName);
+            if (parameterValue.isNull()) {
+                continue;
+            }
+
             try {
-                property.parseValues(entity, Values.of(ctx.getParameters(propertyName)));
+                property.parseValues(entity,
+                                     Values.of(parameterValue.get(List.class,
+                                                                  Collections.singletonList(parameterValue.get()))));
```

Comment:
this looks weird, maybe long term we should look at properties only parsing a Value and the properties that actually support multiple values to look for a list inside of the value

Reply:
Yes, although this would probably require quite a lot of changes in sirius-db